### PR TITLE
Update ADR-00002 Personas related to Managed Services.md

### PR DESCRIPTION
**Pull request by bkulaga9**
QE persona was adjusted. QE manager persona added.

### CODE REVIEW

**Comment by bkulaga9** [comment]
When we reviewed the personas, we believe a crucial persona is missing - Product security.

**Comment by brizrobbo** [comment]
With respect to this comment @bkulaga9 : `When we reviewed the personas, we believe a crucial persona is missing - Product security.`

I agree. How would feel about constructing a draft of what that persona would be like and submitting as another commit (i.e. an update)?

**Comment by bkulaga9** [comment]
@brizrobbo , I sure could do that. However, we should have someone from security involved in sig-sre or provide their opinion/consultation. I don't want to jot something down that will not be true. What do you think?

File: ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md [hunk 1]
```diff
@@ -62,9 +62,15 @@ Also, it is important to acknowledge that there are developers who are also func
 
 The Developers and SREs do not work in isolation. To obtain the economies of scale that are necessary, and to make sure that work is not duplicated, they need to conform to the 'systems (including processes)' and 'architecture' that best suits the business goal. This is the primary role of this persona. They provide the "guard-rails" for the developers, and help keep the services aligned with accepted strategic technical direction. They also have a key role in developing best practices, and coaching teams to avoid anti-patterns.
 
+#### QE Manager
+
+Much like the engineering manager, QE managers need to help cross-functional team relationships as well as drive the work related to creating transparency around functional quality and that higher-level business requirements continue to perform as needed.  This persona should also help the engineering teams to drive conversations around defects management in general, customer tickets, usability concerns, along with metrics around the testing effort related to the various environments in play.
+
+#### Product Security
+
```
Comment (mmazur) [inline ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md]:
```suggestion
```

As someone suggested, this should be added in a separate PR.

File: ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md [hunk 2]
```diff
@@ -62,9 +62,15 @@ Also, it is important to acknowledge that there are developers who are also func
...
 #### Quality Engineer
 
-Quality Engineering (QE) installs and manages instances of the product primarily for the purpose of bug and regression testing, informing Developers of reliability issues and grounding confidence for the other personas that the product functions as intended in the subset of situations covered by QE. SREs need to be able to trust what is being deployed to Production.
+QE is responsible for developing, maintaining and delivering various kinds of tests needed for ensuring service quality. QE is also responsible for providing reports and metrics so that any other personas can check them and react to them. QE covers the service quality through the whole service life-cycle such as during development, during release process and testing in production
```
Comment (mmazur) [inline ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md]: 1. "developing, maintaining… etc. of tests" – I'm missing "performing". Quality engineers do perform various checks themselves.
2. > QE is also responsible for providing reports and metrics so that any other personas can check them and react to them.
    * "reports and metrics" on what? Something like "product and service quality" I'm guessing.
    * Also I'd maybe end with "and make decisions based on them". The interactions I had with QE where that they were like a "gate".
3. There's a dot missing at the very end.

**Review by mmazur — CHANGES_REQUESTED** [review]

File: ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md (hunk 2 above)
Comment (bkulaga9) [reply ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md]: @mmazur please check the verbiage now

**Comment by brizrobbo** [comment]
> @brizrobbo , I sure could do that. However, we should have someone from security involved in sig-sre or provide their opinion/consultation. I don't want to jot something down that will not be true. What do you think?

You could add some suggested text here, and then we can see if we can find some prodsec folks to review. @jeremyeder do you have any suggestions of who to involve? Or, maybe it would be a better idea to keep this idea as a separate PR and just concentrate on adding the "QE" changes in? We can push these "QE" changes through quite quickly I expect, and then concentrate on adding a security persona. WDYT @bkulaga9 ?

**Comment by bkulaga9** [comment]
> You could add some suggested text here, and then we can see if we can find some prodsec folks to review. @jeremyeder do you have any suggestions of who to involve? Or, maybe it would be a better idea to keep this idea as a separate PR and just concentrate on adding the "QE" changes in? We can push these "QE" changes through quite quickly I expect, and then concentrate on adding a security persona. WDYT @bkulaga9 ?

@brizrobbo I agree to have QE changes only for this PR. However, we should not leave security out of the picture, they are an integral part of the development lifecycle.

File: ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md [hunk 3]
```diff
@@ -62,9 +62,13 @@ Also, it is important to acknowledge that there are developers who are also func
 
 The Developers and SREs do not work in isolation. To obtain the economies of scale that are necessary, and to make sure that work is not duplicated, they need to conform to the 'systems (including processes)' and 'architecture' that best suits the business goal. This is the primary role of this persona. They provide the "guard-rails" for the developers, and help keep the services aligned with accepted strategic technical direction. They also have a key role in developing best practices, and coaching teams to avoid anti-patterns.
 
+#### QE Manager
+
+Much like the engineering manager, QE managers need to help cross-functional team relationships as well as drive the work related to creating transparency around functional quality and that higher-level business requirements continue to perform as needed.  This persona should also help the engineering teams to drive conversations around defects management in general, customer tickets, usability concerns, along with metrics around the testing effort related to the various environments in play.
```
Comment (aditya-konarde) [inline ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md]:
```suggestion
Much like the Engineering manager, QE managers need to help cross-functional team relationships as well as drive the work related to creating transparency around functional quality and that higher-level business requirements continue to perform as needed.  This persona should also help the engineering teams to drive conversations around defects management in general, customer tickets, usability concerns, along with metrics around the testing effort related to the various environments in play.
```

**Review by aditya-konarde — APPROVED** [review]
Suggested a fix to a nit on capitalization, otherwise looks good to me! :)

**Comment by brizrobbo** [comment]
Hey @lisa . See the assign to @david-martin  previous. That should be you instead?

**Comment by brizrobbo** [comment]
@mmazur If you are satisfied with @bkulaga9 's change previous, could you please approve?

File: ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md [hunk 4]
```diff
@@ -62,9 +62,13 @@ Also, it is important to acknowledge that there are developers who are also func
 
 The Developers and SREs do not work in isolation. To obtain the economies of scale that are necessary, and to make sure that work is not duplicated, they need to conform to the 'systems (including processes)' and 'architecture' that best suits the business goal. This is the primary role of this persona. They provide the "guard-rails" for the developers, and help keep the services aligned with accepted strategic technical direction. They also have a key role in developing best practices, and coaching teams to avoid anti-patterns.
 
+#### QE Manager
+
+Much like the Engineering manager, QE managers need to help cross-functional team relationships as well as drive the work related to creating transparency around functional quality and that higher-level business requirements continue to perform as needed.  This persona should also help the engineering teams to drive conversations around defects management in general, customer tickets, usability concerns, along with metrics around the testing effort related to the various environments in play.
```
Comment (mmazur) [inline ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md]:
```suggestion
Much like the Engineering Managers, QE Managers need to help cross-functional team relationships as well as drive the work related to creating transparency around functional quality and that higher-level business requirements continue to perform as needed.  This persona should also help the engineering teams to drive conversations around defects management in general, customer tickets, usability concerns, along with metrics around the testing effort related to the various environments in play.
```

File: ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md (hunk 2 above)
Comment (mmazur) [reply ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md]: Yup, +1 from me.

**Review by mmazur — APPROVED** [review]

**Comment by mmazur** [comment]
@brizrobbo do you have any comments to the current change? If not, I'll merge.

**Comment by brizrobbo** [comment]
LGTM @mmazur
